### ISSUE TITLE
clean up process names

### ISSUE THREAD

**Issue by saskiad** [AllenNeuralDynamics/aind-data-schema-models#48]
Once QualityControl is merged into aind-data-schema:

remove
MANUAL_ANNOTATION = "Manual annotation"
QUALITY_CONTROL_AND_ASSESSMENT = "Quality control and assessment"
from ProcessName Model
